governance/proposals/config.js

```diff
@@ -0,0 +1,11 @@
+const ARB_TOKEN_ADRESS_ON_L2 = '0x912CE59144191C1204E64559FE8253a0e49E6548' // ARB TOKEN ADDRESS ON ARBITRUM ONE
```

Comment:
We already have that in the networks package, please is that!

Reply:
Got it